src/timezone.ts

```diff
@@ -41,14 +41,14 @@ export const timezone = async (message: Message) => {
 
       embed.addField(mentioned, proposedZone!.zone);
 
-      availableZones.forEach((item) =>
-        embed.addField(
-          `${item.zone} (${item.abbr})`,
-          initial.tz(item.zone).format('h:mma z'),
-          true
-        )
-      );
-
+      let zones = '';
+      availableZones.forEach((item) => {
+        zones = zones
+          .concat(`**${item.zone} (${item.abbr})**: `)
+          .concat(`${initial.tz(item.zone).format('h:mma z')}`)
+          .concat('\n');
+      });
+      embed.addField('\u200b', zones);
```

Comment:
Probably not as readable, but it could be written like...

```ts
embed.addField(
   '\u200b',
   availableZones.map((zone) => zones
          .concat(`**${item.zone} (${item.abbr})**: `)
          .concat(`${initial.tz(item.zone).format('h:mma z')}`)
          .concat('\n'))
);
```

Reply:
We can perhaps implement this in the next PR(cleaning up the timezones further)👍